pandas/core/dtypes/missing.py

```diff
@@ -408,6 +408,7 @@ def array_equivalent(left, right, strict_nan=False):
 
     # Object arrays can contain None, NaN and NaT.
     # string dtypes must be come to this path for NumPy 1.7.1 compat
+    # TODO: remove old numpy compat code (or comment)
```

Comment:
In this case, I'm not sure what's to be done here (expand below; no numpy code to be found)

Reply:
remove this comment

Reply:
I don't think that "for NumPy 1.7.1 compat" should be left standing, but ok.